sentry-android-core/src/main/java/io/sentry/android/core/SentryLogcatAdapter.java

```diff
@@ -0,0 +1,109 @@
+package io.sentry.android.core;
+
+import android.util.Log;
+import io.sentry.Breadcrumb;
+import io.sentry.Sentry;
+import io.sentry.SentryLevel;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This class replaces {@link android.util.Log} with its own implementations which creates a {@link
+ * io.sentry.Breadcrumb} for each log. It only replaces log functions that meet a minimum level set
+ * by the user on the Sentry Android Gradle Plugin.
+ */
+@ApiStatus.Internal
```

Comment:
Dunno if it makes sense to make it public, people might want to still use it instead of `android.util.Log`, but without the gradle plugin, what do you think @markushi @buenaflor ?

Reply:
IMHO I wouldn't 😅 Since the config for `minLogLevel` lives within the gradle plugin, people calling the `SentryLogcatAdapter` will always generate breadcrumbs, even for e.g. verbose logging